File: integrated-project/mindboost-javafx/src/main/java/com/mindboost/app/model/Score.java
package com.mindboost.app.model;

import jakarta.persistence.*;

@Entity
@Table(name = "score")
public class Score {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    @Column(name = "user_id", nullable = false)
    private Integer userId;

    @Column(name = "general_test_id", nullable = false)
    private Integer generalTestId;

    @Column(name = "total_score", nullable = false)
    private Integer totalScore;

    @Column
    private Integer percentage;

    public Integer getId() {
        return id;
    }

    public Integer getUserId() {
        return userId;
    }

    public void setUserId(Integer userId) {
        this.userId = userId;
    }

    public Integer getGeneralTestId() {
        return generalTestId;
    }

    public void setGeneralTestId(Integer generalTestId) {
        this.generalTestId = generalTestId;
    }

    public Integer getTotalScore() {
        return totalScore;
    }

    public void setTotalScore(Integer totalScore) {
        this.totalScore = totalScore;
    }

    public Integer getPercentage() {
        return percentage;
    }

    public void setPercentage(Integer percentage) {
        this.percentage = percentage;
    }
}

File: symfony/src/Entity/Score.php
<?php

namespace App\Entity;

use App\Repository\ScoreRepository;
use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity(repositoryClass: ScoreRepository::class)]
#[ORM\Table(name: 'score')]
class Score
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column]
    private ?int $id = null;

    #[ORM\Column(name: 'user_id')]
    private ?int $userId = null;

    #[ORM\Column(name: 'general_test_id')]
    private ?int $generalTestId = null;

    #[ORM\Column(name: 'totalScore')]
    private ?int $totalScore = null;

    #[ORM\Column(nullable: true)]
    private ?int $percentage = null;

    public function getId(): ?int { return $this->id; }
    public function getUserId(): ?int { return $this->userId; }
    public function setUserId(int $userId): static { $this->userId = $userId; return $this; }
    public function getGeneralTestId(): ?int { return $this->generalTestId; }
    public function setGeneralTestId(int $generalTestId): static { $this->generalTestId = $generalTestId; return $this; }
    public function getTotalScore(): ?int { return $this->totalScore; }
    public function setTotalScore(int $totalScore): static { $this->totalScore = $totalScore; return $this; }
    public function getPercentage(): ?int { return $this->percentage; }
    public function setPercentage(?int $percentage): static { $this->percentage = $percentage; return $this; }
}